### PR TITLE
Workflow support

### PR DESCRIPTION
**Pull request by jglick**
@reviewbybees

### CODE REVIEW

**Comment by amuniz** [comment]
:bee: (there are some backward compatibility issues, but I guess it does not matter in this case (?)

File: src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildNow.java [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 
     @Override
     public <T> String getText(Message<T> message, T t) {
-        if (message== AbstractProject.BUILD_NOW_TEXT) {
-            AbstractProject i = (AbstractProject) t;
-
-            BuildNowTextProperty bt = (BuildNowTextProperty) i.getProperty(BuildNowTextProperty.class);
+        if (message == ParameterizedJobMixIn.BUILD_NOW_TEXT) {
+            Job<?,?> p = (Job) t;
+            BuildNowTextProperty bt = p.getProperty(BuildNowTextProperty.class);
```
Comment (tfennelly) [inline src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildNow.java]: Not new to this code, but shouldn't there be a `null` check on `p`?

**Comment by tfennelly** [comment]
:bee:

File: src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildNow.java (hunk 1 above)
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/customizebuildnow/AlternateBuildNow.java]: I do not think so.

**Comment by jglick** [comment]
> backward compatibility issues

You mean on the constructor? No one cares.

**Comment by amuniz** [comment]
Yeah, and the removed `public boolean isSpecified()`

**Comment by jglick** [comment]
In case you were wondering why I am patching this relatively obscure plugin, it is because I wanted an interactive test case for [JENKINS-30519](https://issues.jenkins-ci.org/browse/JENKINS-30519).

**Comment by amuniz** [comment]
Yeah, I was :smile:

**Pull request merged**